apmpackage/apm/data_stream/internal_metrics_interval/manifest.yml

```diff
@@ -0,0 +1,36 @@
+title: APM internal metrics
+type: metrics
+dataset: apm.internal-interval
```

Comment:
[`data_stream.dataset`](https://www.elastic.co/guide/en/ecs/current/ecs-data_stream.html#field-data-stream-dataset) is not allowed to contain "-", as that's the separator used for identifying the data stream type, dataset, and namespace.

Reply:
> I would strongly prefer if we introduced transaction and service metric data streams.

I’m good with separating out data streams for metrics that we’re downsampling - transaction, service, and service destination metrics.

Reply:
So we'd have 1 internal metric data stream (`apm-metrics.internal`), and 3 "aggregation" metrics data streams (1 per aggregated interval) then.